pkg/policy/api/egress_test.go

```diff
@@ -308,6 +308,29 @@ func (s *PolicyAPITestSuite) TestIsLabelBasedEgress(c *C) {
 				}
 			},
 		},
+		{
+			name: "rule-with-icmp",
+			setupArgs: func() args {
+				return args{
+					&EgressRule{
+						ICMP: ICMPRules{
+							{
+								Types: []ICMPType{
+									{
+										Type: 0,
```

Comment:
It's probably better to assert against non-zero values, because it would give more confidence that the code has worked properly, as [Go will zero all values](https://tour.golang.org/basics/12) during linking. If the implementation is buggy, it's possible that the test could still pass because the values would be zeroed.

Reply:
OK I'll set the other number :)